src/webui/utils/api.js

```diff
@@ -1,49 +1,84 @@
 import storage from './storage';
+import {HTTP_STATUS} from '../../../src/lib/constants';
 
-class API {
-  request(url, method = 'GET', options = {}) {
-      if (!window.VERDACCIO_API_URL) {
-        throw new Error('VERDACCIO_API_URL is not defined!');
-      }
+/**
+ * CHECK FOR BASE API URL
+ */
+if (!window.VERDACCIO_API_URL) {
+  throw new Error('VERDACCIO_API_URL is not defined!');
+}
+const BASE_URL = window.VERDACCIO_API_URL;
```

Comment:
I'd move this above the previous condition and use `BASE_URL` with the `if` condition. So, easy to change in the future.